packit_service/worker/handlers/bodhi.py

```diff
@@ -167,3 +171,51 @@ def notify_user_about_failure(self, body: str) -> None:
             + f"\n\n---\n\n*Get in [touch with us]({CONTACTS_URL}) if you need some help.*",
             comment_to_existing=body,
         )
+
+
+@configured_as(job_type=JobType.bodhi_update)
+@reacts_to(event=KojiBuildEvent)
+class CreateBodhiUpdateHandler(
+    BodhiUpdateHandler,
+    RetriableJobHandler,
+    GetKojiBuildEventMixin,
+    GetKojiBuildDataFromKojiBuildEventMixin,
+):
+    """
+    This handler can create a bodhi update for successful Koji builds.
+    """
+
+    task_name = TaskName.bodhi_update
+
+    @staticmethod
+    def get_checkers() -> Tuple[Type[Checker], ...]:
+        """We react only on finished builds (=KojiBuildState.complete)
+        and configured branches.
+        """
+        logger.debug("Bodhi update will be re-triggered via dist-git PR comment.")
+        return (IsKojiBuildCompleteAndBranchConfiguredCheckEvent,)
+
+
+@configured_as(job_type=JobType.bodhi_update)
+@reacts_to(event=PullRequestCommentPagureEvent)
+@run_for_comment(command="create-update")
+class RetriggerBodhiUpdateHandler(
+    BodhiUpdateHandler, GetKojiBuildDataFromKojiServiceMixin
+):
+    """
+    This handler can re-trigger a bodhi update if any successfull Koji build.
+    """
+
+    task_name = TaskName.retrigger_bodhi_update
+
+    @staticmethod
+    def get_checkers() -> Tuple[Type[Checker], ...]:
+        """We react only on finished builds (=KojiBuildState.complete)
+        and configured branches.
+        """
+        logger.debug("Bodhi update will be re-triggered via dist-git PR comment.")
+        return (
+            HasAuthorWriteAccess,
+            IsAuthorAPackager,
+            IsKojiBuildCompleteAndBranchConfiguredCheckService,
+        )
```

Comment:
Will Packit provide any feedback back to the user if one of these checkers fails?

Reply:
You are right, there is no feedback for the users when checks fail. For none of the them.
If you agree I will create a new card to add the feedback, I think there is some work on it. Right now there is no `BodhiUpdateJobHelper` class and for the other services is this kind of helper class that handles the feedback.

Reply:
agreed! we should absolutely have something to comment back so users know what's happening in Packit

Reply:
> agreed! we should absolutely have something to comment back so users know what's happening in Packit

https://github.com/packit/packit-service/issues/1747